test/internal/cleanup_test.go

```diff
@@ -85,7 +85,7 @@ func TestCleanup(t *testing.T) {
 	t.Logf("projects:\n%s\n", resp)
 	for _, project := range projects.GetResults() {
 		projectID := project.GetId()
-		if projectID == os.Getenv("MONGODB_ATLAS_PROJECT_ID") {
+		if projectID == ProfileProjectID() || projectID == os.Getenv("MONGODB_ATLAS_PROJECT_ID") {
```

Comment:
or gov, we override the project name to `__e2e_gov`, so we were probably not skipping it correctly